config/apiserver/apiserver.template.yaml

```diff
@@ -109,13 +109,13 @@ objects:
         volumes:
         - name: apiserver-certs
           secret:
-            secretName: ${SVC_NAME}
```

Comment:
Is this file still used for anything?

Reply:
i haven't killed off the current hack/deploy-apiserver.sh, so it still uses the template